_posts/2023-04-02-a-journey-with-apache-arrow-part-1.md

```diff
@@ -0,0 +1,382 @@
+---
+layout: post
+title: "A journey with Apache Arrow (part 1)"
```

Comment:
How about adding something related to OpenTelemetry to the title?
https://arrow.apache.org/ is the Apache Arrow web site. So all contents are related to Apache Arrow. In the context, the current title may be too generic. If we can add some contexts in the title, it may help readers.